contenteditable/plaintext-only.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>contenteditable="plaintext-only" should exist</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="txt" contenteditable="plaintext-only"></div>
+<script>
+test(function() {
+    const txt = document.getElementById('txt');
+    assert_true(txt.isContentEditable);
+    assert_equals(txt.contentEditable, 'plaintext-only');
+});
```

Comment:
We probably want another test where the 'plaintext-only' is set dynamically too. So, like, create a new div, check the initial state of `isContentEditable` being false, and then setting  `.contentEditable = 'plaintext-only'` and make sure it doesn't throw.

Reply:
After the test above, you can ask web driver to click() it. Then you can send keys to it. 

What we can also do is copy HTML into the clipboard via the clipboard API... we should see if there is some way to paste. I'm not sure if there is, but would be neat to find out.